src/Jackalope/ObjectManager.php

```diff
@@ -869,6 +869,7 @@ public function save()
                 if ($item->isModified() || $item->isMoved()) {
                     $item->confirmSaved();
                 }
+                $item->setClean();
```

Comment:
i think we did not do this so that auto generated things would be found when reloading (jackrabbit might autocreate some things)